src/Web3/Util.elm

```diff
@@ -193,6 +193,12 @@ fromWei unit amount =
                 (\i -> "")
 
 
+bigIntToWei : EthUnit -> BigInt -> BigInt
+bigIntToWei unit amount =
+    toWei unit (toString amount)
```

Comment:
Nice and simple, when people start needing to call this 1000 times a second we'll relook at efficiency :joy:

Reply:
I made a possibly more efficient one just for fun 😄 . See below.

Reply:
Oh my! Yes, that is broken!

Reply:
Elm types don't catch everything, haha.

My original `toWei` was very efficient, but it had that big helper function. And yes the `BigInt.pow` is all that it needed.